iree/turbine/kernel/_support/tracing.py

```diff
@@ -111,6 +131,12 @@ def proxy(self, node: fx.Node) -> fx.Proxy:
                 node.meta["dtype"] = t
                 node.meta["symbolic_type"] = []
                 self.current_arg_id += 1
+            elif issubclass(t, SymbolBind) and node.op == "placeholder":
```

Comment:
Could you instead do
```
elif issubclass(t, SymbolBind):
      assert node.op == "placeholder" ...
```
otherwise SymbolBinds on non-placeholders will silently fail.

Reply:
done